docs/source/quickinstall.rst

```diff
@@ -8,39 +8,52 @@ This quick install guide outlines the basic steps needed to install OpenMC on
 your computer. For more detailed instructions on configuring and installing
 OpenMC, see :ref:`usersguide_install` in the User's Manual.
 
-----------------------------------------
-Installing on Linux/Mac with conda-forge
-----------------------------------------
-
-`Conda <https://conda.io/en/latest/>`_ is an open source package management
-system and environment management system for installing multiple versions of
-software packages and their dependencies and switching easily between them. If
-you have `conda` installed on your system, OpenMC can be installed via the
+--------------------------------------------------
+Installing on Linux/Mac with Mamba and conda-forge
+--------------------------------------------------
+
+`Conda <https://conda.io/en/latest/>`_ and `Mamba <https://mamba.readthedocs.io/en/latest/>`_
+are an open source package management systems and environments management
+system for installing multiple versions of software packages and their
+dependencies and switching easily between them. If you have `conda` installed
+on your system, OpenMC can be installed via the
 `conda-forge` channel. First, add the `conda-forge` channel with:
 
 .. code-block:: sh
 
     conda config --add channels conda-forge
 
+Then install `mamba`, which will later be used to install OpenMC in the conda environment.
+
+.. code-block:: sh
+
+    conda install mamba
+
 To list the versions of OpenMC that are available on the `conda-forge` channel,
 in your terminal window or an Anaconda Prompt run:
 
 .. code-block:: sh
 
-    conda search openmc
+    mamba search openmc
 
-OpenMC can then be installed with:
+First create and activate a new conda enviroment called `openmc-env` in which to install OpenMC.
 
 .. code-block:: sh
 
-    conda create -n openmc-env openmc
+    conda create -n openmc-env
```

Comment:
It looks like `mamba` also has a `create` command, so I think the right way to do this is to install mamba into the base environment and then use `mamba create`.

Reply:
Yes, there is a create command, but I tested doing that and it did not work (same fail as if using conda to install). I don't know if maybe it uses conda instead when running that?

Reply:
I had this same dilemma, so I'm keen to see what you chose here. In the end I went for mamba in the newly created environment like this https://paramak--243.org.readthedocs.build/en/243/install.html#install-mamba

Reply:
I am going to update the instructions to follow a similar order (basically install mamba in the new env instead of base which is what is currently suggested).